Sem_1/Zad1.py

```diff
@@ -0,0 +1,20 @@
+""""
+Задача 2: Найдите сумму цифр трехзначного числа.
+*Пример:*
+
+123 -> 6 (1 + 2 + 3)
+100 -> 1 (1 + 0 + 0)
+"""
+a = input('Введите целое 3-значное число: ')
+if len(a)==3:
+    sum = 0
+    a=int(a)
+    while a>0:
+        x=a%10
+        sum+=x
```

Comment:
Можно сократить без потери читаемости
```suggestion
        sum += a % 10
```